<?xml version = "1.0"?>
<launch>
    <!-- Define PID gains as parameters -->
    <param name="Kp_m1" type="double" value="1.0"/>
    <param name="Ki_m1" type="double" value="1.0"/>
    <param name="Kd_m1" type="double" value="0.01"/>
     <!-- Define PID gains as parameters -->
    <param name="Kp_m2" type="double" value="1.0"/>
    <param name="Ki_m2" type="double" value="1.0"/>
    <param name="Kd_m2" type="double" value="0.01"/>
    <!-- Define robot parameters all units are in m, m/s -->
    <param name="wheel_radius" type="double" value=".20"/> 
    <param name="max_vel" type="double" value="1.54"/>
    <param name="wheel_separation" type="double" value="0.83"/> 
    <!-- Include your node launch -->
    <!--node pkg="agribot_ctrl" type="AgribotPid" name="AgribotPid" output="screen"/-->
    <!--node pkg="agribot_ctrl" type="AgribotOdomEnc" name="AgribotOdomEnc" output="screen"/-->    
    <node pkg="agribot_ctrl" type="AgribotTwist" name="AgribotTwist" output="screen"/>    
    <node pkg="agribot_ctrl" type="AgribotVel" name="AgribotVel" output="screen"/>
    <!-- <node name="arduino_node2" pkg="rosserial_python" type="serial_node.py" output="screen">
      <param name="port" value="/dev/ttyACM0"/>
      <param name="baud" value="38400"/>
    </node> -->
</launch>
